packages/core/build/tokens.ts

```diff
@@ -320,13 +320,13 @@ const typography = {
     500: token(color.black), // content
   },
   fontWeight: {
-    // Clarity City is limited to 400-500, tokens provide hooks for customization
-    light: token('200'),
+    // Clarity City is limited to a minimum weight of 300 and max weight of 600, tokens provide hooks for customization
+    light: token('300'),
     regular: token('400'),
     medium: token('500'),
-    semibold: token('500'),
-    bold: token('500'),
-    extrabold: token('500'),
+    semibold: token('600'),
```

Comment:
I am curious why do we have 3 typography tokens with the same value?

Reply:
@bbogdanov : I sent you the Slack thread explaining this over Slack. Summary for those not on Slack, it has to do with the way the foundry set up the font. Not all weights are viable so this is a workaround to utilize the strengths of the font we have.